internal/api/middleware.go

```diff
@@ -118,6 +118,10 @@ func (a *API) verifyCaptcha(w http.ResponseWriter, req *http.Request) (context.C
 
 	verificationResult, err := security.VerifyRequest(req, strings.TrimSpace(config.Security.Captcha.Secret), config.Security.Captcha.Provider)
 	if err != nil {
+		if strings.Contains(err.Error(), "request body was not JSON") {
```

Comment:
Can consider eventually swapping to error code or similar in future

Reply:
Yeah the captcha module should be reworked